oam/install_scripts/post-install

```diff
@@ -93,53 +95,65 @@ if [ -f libstdc++.so.6.0.14 ]; then
 fi
 
 # softlink for libperl.sp, used by cplogger
-$SUDO ln -s /usr/lib64/perl5/CORE/libperl.so /usr/lib64/libperl.so >/dev/null 2>&1
-
-#setup profile files
-profileFileAlias=/etc/profile.d/columnstoreAlias.sh
-profileFileEnv=/etc/profile.d/columnstoreEnv.sh
+ln -s /usr/lib64/perl5/CORE/libperl.so /usr/lib64/libperl.so >/dev/null 2>&1
 
 if [ $installdir != "/usr/local/mariadb/columnstore" ]; then
         sed -i -e s@/usr/local/mariadb/columnstore@$installdir@g $installdir/bin/columnstoreAlias
 fi
 
 if [ $user != "root" ]; then
-	sudo rm -f $profileFileEnv
-	sudo rm -f $profileFileAlias
-	
-	sudo touch $profileFileEnv
-	sudo chmod 666 $profileFileEnv
-	egrep -qs 'MariaDB Columnstore Non-Root' ${profileFileEnv}
+	if [ -f $prefix/.bash_profile ]; then		
+		profileFile=$prefix/.bash_profile
+	elif [ -f $prefix/.profile ]; then		
+		profileFile=$prefix/.profile
+	else
+		profileFile=$prefix/.bashrc
+		touch $prefix/.bashrc
+	fi
+		
+	if [ ! -f ${profileFile}_backup ]; then
+	    cp $profileFile ${profileFile}_backup
+	fi
+
+	egrep -qs 'MariaDB Columnstore Non-Root Environment Variables' ${profileFile}
 	rc1=$?
 	if [ $rc1 -ne 0 ]; then
-        	sudo echo " " >> ${profileFileEnv}
-		sudo echo "# MariaDB Columnstore Non-Root Environment Variables" >> ${profileFileEnv}
-        	sudo echo "export COLUMNSTORE_INSTALL_DIR=$COLUMNSTORE_INSTALL_DIR" >> ${profileFileEnv}
-        	sudo echo "export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$COLUMNSTORE_INSTALL_DIR/lib:$COLUMNSTORE_INSTALL_DIR/mysql/lib" >> ${profileFileEnv}
-        	. ${profileFileEnv}
+        	echo " " >> ${profileFile}
+		echo "# MariaDB Columnstore Non-Root Environment Variables" >> ${profileFile}
+        	echo "export COLUMNSTORE_INSTALL_DIR=$COLUMNSTORE_INSTALL_DIR" >> ${profileFile}
+        	echo "export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$COLUMNSTORE_INSTALL_DIR/lib:$COLUMNSTORE_INSTALL_DIR/mysql/lib" >> ${profileFile}
 	fi
 	
-	sudo /bin/cp -f $installdir/bin/columnstoreAlias $profileFileAlias
-	sudo chmod 666 $profileFileAlias
+	egrep -qs 'MariaDB Columnstore Non-Root Alias Variables' ${profileFile}
+	rc1=$?
+	if [ $rc1 -ne 0 ]; then
+        	echo " " >> ${profileFile}
+		echo "# MariaDB Columnstore Non-Root Alias Variables" >> ${profileFile}
+		cat $installdir/bin/columnstoreAlias >> ${profileFile}
+	fi
+
+	#source the file
+	. ${profileFile}
 else
-	/bin/cp -f $installdir/bin/columnstoreAlias $profileFileAlias
-	chmod 644 $profileFileAlias
+	profileFile="/etc/profile.d/columnstoreAlias.sh"
```

Comment:
We redefine profileFile here; not sure we want to do that.  Looks like it's used later.